src/ng/compile.js

```diff
@@ -3101,6 +3152,33 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
         }
       });
 
+      function recordChanges(key, currentValue, previousValue) {
+        if (isFunction(destination.$onChanges) && currentValue !== previousValue) {
+          // If we have not already scheduled the top level onChangesQueue handler then do so now
+          if (!onChangesQueue) {
+            scope.$$postDigest(flushOnChangesQueue);
```

Comment:
Hi @doberman can you clarify what you mean here?
Are you saying that the use of EvalAsync is not needed any more?

I did consider piggybacking on the rootscope TTL but since it is not actually related to the digest cycle I  thought we should keep them independent